docs/manage/requesting_data_as_a_collaborator.md

```diff
@@ -1,63 +1,68 @@
-# Requesting Data as a Collaborator
+# Requesting & Using Data as a Collaborator
 ---
+## Sourcing Data
+
+The **Transportation Secure Data Center (TSDC)**  hosts data collected by OpenPATH during a variety of surveys.  This data can be used to replicate previous study findings, generate new visualizations, or simply to explore the platform's capabilites.  To request data from a specific program, please visit the TSDC [website](https://www.nrel.gov/transportation/secure-transportation-data/index.html).
+
+## Working With Data  ##
+
+After requesting data from TSDC, you should receive a "mongodump" file -- a collection of data, archived in `.tar.gz` format.  Here are the broad steps you need to take in order to work with this data:
```

Comment:
The TSDC will not provide mongodumps. The TSDC will provide access to the data in csv files/postgres database. The mongodump is currently *only* available for internal use.